core/integration/git_test.go

```diff
@@ -438,6 +438,7 @@ func (GitSuite) TestServiceStableDigest(ctx context.Context, t *testctx.T) {
 			WithMountedDirectory("/repo", c.Git(url, dagger.GitOpts{
 				ExperimentalServiceHost: svc,
 			}).Branch("main").Tree()).
+			WithDefaultArgs([]string{"/bin/sh"}).
```

Comment:
nit: this command isn't a service - we should explicitly make it a command like `sleep` or something, instead of one that would just immediately exit if it was started.

(nit because the service isn't really started, so it doesn't matter)